spec/factories/item_season.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :item_season do
+    produce_item_id { build(:produce_item) }
```

Comment:
**suggestion:** I'm kind of surprised FactoryBot is smart enough to fetch the id here, since you are building an entity, not accessing an id. A more explicit way to do that would be:
```suggestion
    produce_item { build(:produce_item) }
```